src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -406,6 +406,12 @@
 type: command
 short-summary: Queues a quick run providing streamed logs for an Azure Container Registry.
 examples:
+  - name: Queue a contextless run.
+    text: >
+        az acr run -r MyRegistry --cmd bash /dev/null
+  - name: Queue a run from standard input. Either 'Ctrl + z' or 'Ctrl + d' terminates the input stream.
```

Comment:
'Ctrl + z' (Windows) or 'Ctrl + d' (Linux)

Reply:
Can you also add a pipeline example, like cat task.yaml | az acr run -r MyRegistry -f - .

Reply:
Done